tests/test_prove_stat_validation.py

```
@@ -0,0 +1,197 @@
+
+def test_type_i_normal_noise():
+    """IID normal noise -> rejection rate ~= alpha."""
+    rate = _rejection_rate(lambda rng: [rng.gauss(0, 1) for _ in range(SERIES_LEN)])
+    assert 0.02 <= rate <= 0.09, f"Type I too far from alpha: {rate:.3f}"
+
+
+def test_type_i_integer_counts():
+    """Poisson-ish integer counts (realistic correction data) -> ~= alpha."""
+    weights = [8, 6, 5, 4, 3, 2, 2, 1, 1, 1, 1]
+
+    def gen(rng):
```

Ruff (0.15.10)

[warning] Missing return type annotation for private function `gen`

(ANN202)